### DISCUSSION TITLE
How to include chapters from subdirectories in quarto book project?

### DISCUSSION THREAD

**Discussion by chrizz89** [quarto-dev/quarto-cli#1761] [Q&A] [answered]
I would like to organize my quarto book project by putting the *qmd files for each chapter in a separate sub directory. 

I tried the following in the _quarto.yml
```
book:
  chapters:
    - subdir/chapter.qmd
```
but this does not work. Is it possible to organize my project in that way?

**Comment by jjallaire** [comment, marked as answer]
This does indeed work. Here is a simple git repo demonstrating: https://github.com/jjallaire/subdir-chapters/

**Reply by chrizz89** [reply]
OK wow, this is emberassing: I simply misspelled the directory name. It does work now. Thanks for your answer!

**Reply by jgunstone** [reply]
hi there, is this sort of syntax: 

```yaml
book:
  chapters:
    - subdir/*.qmd
```

to recursively add all files in that dir as chapters?

**Reply by mcanouil** [reply]
If you tried, did not work, and it's not documented, then the answer is no.

**Reply by jgunstone** [reply]
understood. this functionality is supported for websites so I thought I might have been missing something

- [x] tried
- [x] did not work
- [x] not documented

**Reply by mcanouil** [reply]
Now, you could open a feature request explaining why it could be useful to have this: https://github.com/quarto-dev/quarto-cli/discussions/new?category=feature-requests (if no issues/discussions are already open about this)

**Reply by jgunstone** [reply]
it looks like someone has in-fact already requested this feature
https://github.com/quarto-dev/quarto-cli/discussions/4874
sorry for bad git-ing!
